### ISSUE TITLE
GoGo Shell Support Fixed

### ISSUE THREAD

**Issue by vaiism** [payara/Payara#3450]
# Description #
----------

Using the _gogo_ shell within asadmin no longer works (and trying to attempt remote access doesn't either).

The ability to utilize the gogo shell (that was part of GF4/5, etc.) helps tremendously for more advanced OSGi debugging, as well as ability to add enhancements, such as the ECF OSGi command provider, which allows better debugging of enabling remote services (within Payara). 

The main felix shell still exists (_asadmin osgi lb_), but the main gogo shell does not anymore.

## Expected Outcome

I expected gogo to work as it had previously _(asadmin osgi-shell)_.

## Current Outcome

The _osgi-shell_ comes up, but there is no output:

````
$ ~/servers/payara/payara5/bin/asadmin osgi-shell
Use "exit" to exit and "help" for online help.
gogo$ help
gogo$
gogo$ lb
gogo$
````

## Steps to reproduce (Only for bug reports) 

Once the domain is up and running, simply access the _osgi-shell_:

`$ asadmin osgi-shell`

This will put you in the _gogo_ shell:

`gogo$`

From here, any previously working gogo commands no longer work, and appear to just return.

## Context (Optional)

We have always utilized the hybrid OSGi/EE paradigm for our apps, and works great. But not having the gogo shell to help debug certain things, and to specifically be able to add debugging additions to the console is hurting quite a bit, particularly when getting remote services working (ECF) from within.

## Environment ##

- **Payara Version**: 5.182
- **Edition**: Full
- **JDK Version**:  1.8.0_111 or 1.8.0_144-b01
- **Operating System**: Linux (RHEL 7) and Mac (10.13.3)
- **Database**: N/A

**Comment by Cousjava** [comment]
Thank you for reporting this issue. I have confirmed it and raided internal issue PAYARA-3319

**Comment by mulderbaba** [comment]
fixed by #3472
